src/Poseidon/CLI/Forge.hs

```diff
@@ -189,13 +194,13 @@ runForge (
     currentTime <- liftIO getCurrentTime
     newNrSNPs <- liftIO $ catch (
         runSafeT $ do
-            (eigenstratIndEntries, eigenstratProd) <- getJointGenotypeData logEnv intersect_ relevantPackages maybeSnpFile
-            let newEigenstratIndEntries = map (eigenstratIndEntries !!) relevantIndices
-
+            (eigenstratIndEntries, eigenstratProd) <- getJointGenotypeData logEnv intersect_ plinkPopMode relevantPackages maybeSnpFile
+            let eigenstratIndEntriesV = eigenstratIndEntries
```

Comment:
What's the meaning of this suffix `V` variable? I remember that I removed that recently. Now it's back. What am I missing here? Is this some clever design pattern? :sweat_smile:

Reply:
Oh, I think that may have creeped in during the merge. I also can't remember. I know I usually use the `V` suffix to denote `Vector`s and distinguish them from lists. Not sure what that has to do with this place. Hmm, happy to simplify this.

Reply:
Done.